### DISCUSSION TITLE
Running a file I've assembled externally

### DISCUSSION THREAD

**Discussion by RobBrooks** [dirkwhoffmann/vAmiga#824] [General]
I'd like to use vAmiga for Amiga development and I'm looking for the path of least resistance to test a file I've just assembled on the Mac using VASM. The faster I can repeat the assemble and test cycle, the better I'll like it. :)
Any suggestions most welcome.

**Comment by nicolasbauw** [comment]
Personally I generate an adf with xdftool in the build script, then slide this file in vAmiga to insert it in df0.
Probably not the best way but I did not find better.

**Comment by RobBrooks** [comment]
Many thanks for the reply Nicolas.
I have resorted to the 'drag the folder into the application window, over the df0 icon' and then hit the soft reset button.
I must admit I didn't realise I could do this as I was relying on the menus to give me the feature rather than fully RTFM'ing as I should have.
While ideally, I'd have an atomic process whereby I could reload the folder as a drive image + soft reset, I'm actually quite happy with this approach.
Many thanks.
